visualdl/julia/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+find_package(JlCxx REQUIRED)
```

Comment:
Is it more appropriate to provide a cmake file(such as cmake/external/JlCxx.cmake) ?

Reply:
To make this package simple enough. I prefer to specify the JlCxx_DIR in the building stage.